app/public/views/userView/data/table/tablePanel.js

```diff
@@ -72,15 +73,7 @@ export default function tablePanel(model, runs, detectors) {
             },
         }, iconDataTransferDownload()),
 
-        h('button.btn.btn-secondary.icon-only-button', {
-            onclick: () => {
-                navigator.clipboard.writeText(url.toString())
-                    .then(() => {
-                    })
-                    .catch(() => {
-                    });
-            },
-        }, iconShareBoxed()),
+        copyLinkButton(model.router.getUrl().toString()),
```

Comment:
toString() not needed

Reply:
But it stays until the next refactor because everywhere else you used it this way. I want to change things like this all at once not to generate unnecessary lines of changed code in a PR that's purpose is simply to **add a link to the BK fill page**.